com.b2international.snowowl.validation.snomed/src/tests/java/com/b2international/snowowl/validation/snomed/AllGenericValidationTests.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2018 B2i Healthcare. All rights reserved.
```

Comment:
Use the OSS license header instead of the proprietary when pushing changes to this repository.